docs/design/operand-deployment-lifecycle-manager.md

```diff
@@ -177,9 +177,7 @@ Suppose IAM Operator has two CRDs: Apikey and Identity:
                 "name": "iam-identity"
               },
               "spec": {
-                "key": "value",
-                "nested": {
```

Comment:
why kill here?

Reply:
The example in step 1 just has the "key": "value" like this for identity:
```
- name: iam
  spec:
    apikey:
      key: value
      nested:
        key: value
    identity:
      key: value
```
Not sure my understanding is correct,in the csv it should like this:
```
        {
          "apiVersion": "iam.operator.ibm.com/v1alpha1",
          "kind": "Identity",
          "metadata": {
            "name": "iam-identity"
          },
          "spec": {
            "key": "value"
          }
        }
```

Reply:
Good catch, thanks.